app/Console/Commands/ConvertSearchable.php

```diff
@@ -0,0 +1,288 @@
+<?php
```

Comment:
Tell me again why not we do it on demand, when Eloquent emits crud events? I got the point of having a command for initial seeding but is it neccessary later?